.prettierignore

```diff
@@ -4,4 +4,5 @@ packages/plugin-vue/dist/
 packages/*/CHANGELOG.md
 LICENSE.md
 .prettierignore
-yarn.lock
\ No newline at end of file
+yarn.lock
+packages/playground/tsconfig-json/has-error/tsconfig.json
```

Comment:
I think it's better to have a `// prettier-ignore` comment in the `tsconfig.json`.

Reply:
unfortunately that does not work. The parser error trips up prettier and it never reads the ignore comment.